### PR TITLE
TELCODOCS-889 Populating TP features for Telco core

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-889]: Document supportability and uncategorized RN items

Version(s):
4.12 RN

Issue:
https://issues.redhat.com/browse/TELCODOCS-889

Link to docs preview:https://54102--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-technology-preview

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mmirecki** [comment]
The `sysctls list` section looks good. For the others please check with their feature owners.

**Comment by zeeke** [comment]
`Multi-network policies for SR-IOV networks` LGTM

**Comment by fedepaol** [comment]
lgtm

**Comment by mburke5678** [comment]
@kquinn1204 A nit; a question. Otherwise LGTM.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1498,11 +1498,25 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
-|Ingress Node Firewall Operator
+|Advertise using BGP mode the MetalLB service from a subset of nodes, using a specific pool of IP addresses
```
[the remark this answers is not in the document]
Comment (kquinn1204) [reply release_notes/ocp-4-12-release-notes.adoc:1501]: That ingress node Operator item looks wrong

**Comment by kquinn1204** [comment]
I reached out to @JoeAldinger about that entry something fishy going on.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-12-release-notes.adoc:1501]: Hmm, I'm not sure what is going on here. It looks like you added a bunch of stuff into the middle of the tech preview table. I would start by rebasing and see if that helps.

**Comment by kquinn1204** [comment]
Hi @mburke5678 and @JoeAldinger I started afresh and this PR https://github.com/openshift/openshift-docs/pull/54400 now replaces https://github.com/openshift/openshift-docs/pull/54102. I may have messed up a previous rebase and merge conflict resolution, so best to recreate IMO.

**Pull request closed without merging**